tests/BUILD.bazel

```diff
@@ -389,3 +389,49 @@ java_fuzz_target_test(
     allowed_findings = ["com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh"],
     target_class = "com.example.SilencedFuzzer",
 )
+
+java_library(
+    name = "OfflineInstrumentedTarget",
+    srcs = ["src/test/java/com/example/OfflineInstrumentedTarget.java"],
+)
+
+genrule(
+    name = "OfflineInstrumentedTargetInstrumented",
+    srcs = [
+        ":libOfflineInstrumentedTarget.jar",
+    ],
+    outs = ["OfflineInstrumentedTargetInstrumented.jar"],
+    cmd = """
+set -euo pipefail
+
+export JAVA_HOME=$$(realpath $(JAVABASE))
+export PATH="$$JAVA_HOME/bin:$$PATH"
+
+jacococli=$$(realpath external/jacococli/file/jacococli.jar)
+java -jar $$jacococli instrument "$<" --dest "$$PWD"
+
+# Copy instrumented jar to expected output destination
+cp $$PWD/$$(basename "$<") $@
+""",
+    target_compatible_with = LINUX_ONLY,
+    toolchains = [
+        "@bazel_tools//tools/jdk:current_java_runtime",
+    ],
+    tools = [
+        "@jacococli//file:jacococli.jar",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+java_fuzz_target_test(
+    name = "OfflineInstrumentedFuzzer",
+    timeout = "short",
+    srcs = ["src/test/java/com/example/OfflineInstrumentedFuzzer.java"],
+    allowed_findings = ["java.lang.IllegalStateException"],
+    target_class = "com.example.OfflineInstrumentedFuzzer",
+    target_compatible_with = LINUX_ONLY,
```

Comment:
Just curious, why does this work on Linux only?

Reply:
To be honest I haven't checked in details. The pipeline failed and I assumed the scipt just couldn't be executed on other systems. But it should work on macOS, shouldn't it? I will check that in detail again.